conans/test/integration/configuration/test_profile_priority.py

```diff
@@ -43,3 +44,30 @@ def test_profile_cache_folder_priority():
 
     c.run("install . -pr=default")
     assert "os=AIX" in c.out
+
+
+def test_profile_cli_priority():
+    c = TestClient()
+    profile1 = textwrap.dedent("""\
+        [settings]
+        os=AIX
+        [conf]
+        user.myconf:myvalue1=1
+        user.myconf:myvalue2=[2]
+        user.myconf:myvalue3={"3": "4"}
```

Comment:
Can you add another key to this dict? Just to ensure the update won't mess with that value

Reply:
Added, fixed a bug with it! Good catch!